### PR TITLE
test: fix pipeline integ tests

### PR DESCRIPTION
**Pull request by efekarakus**
Since we don't prefix the pipeline stack with the project name anymore
the integ tests failed to delete the pipeline stack because it was
expecting the wrong name for the stack.

_By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice._

### CODE REVIEW

**Comment by efekarakus** [comment]
I'm running the tests in my account atm, i'll merge once they pass

File: internal/pkg/deploy/cloudformation/pipeline_integration_test.go [hunk 1]
```diff
@@ -40,12 +40,11 @@ func TestPipelineCreation(t *testing.T) {
 		createMockSecret(t, sm, secretId)
 		projCfClient := awsCF.New(projectSess)
 
-		const pipelineName = "pipepiper"
 		project := archer.Project{
 			Name:      randStringBytes(10),
 			AccountID: projectCallerInfo.Account,
 		}
-		pipelineStackName := project.Name + "-" + pipelineName
+		pipelineStackName := project.Name + "-pipepiper"
```
Comment (SoManyHs) [inline internal/pkg/deploy/cloudformation/pipeline_integration_test.go:47]: wait, don't we need to get rid of the project name here?

Comment (efekarakus) [reply internal/pkg/deploy/cloudformation/pipeline_integration_test.go:47]: this is naming the pipeline "project.Name-pipepiper". If we don't prefix "pipepiper" with some unique id then it's possible that an old test run that failed will cause the integ test to fail

**Comment by efekarakus** [comment]
Tests are passing, merging the change

**Pull request merged**